ios/FluentUI/Bottom Commanding/BottomCommandingController.swift

```diff
@@ -19,6 +19,30 @@ import UIKit
 @objc(MSFBottomCommandingController)
 open class BottomCommandingController: UIViewController {
 
+    /// View controller that will be displayed below the bottom commanding UI.
+    @objc public var contentViewController: UIViewController? {
+        didSet {
+            if isViewLoaded,
+               oldValue != contentViewController {
+                if let oldViewController = oldValue {
+                    oldViewController.willMove(toParent: nil)
+                    oldViewController.view.removeFromSuperview()
+                    oldViewController.removeFromParent()
+                }
+
+                if let newContentViewController = contentViewController {
+                    addChild(newContentViewController)
+                    if let rootCommandingView = rootCommandingView {
+                        let newContentView: UIView = newContentViewController.view
+                        view.insertSubview(newContentView, belowSubview: rootCommandingView)
+                        activateContentViewConstraints(for: newContentView)
+                    }
+                    newContentViewController.didMove(toParent: self)
```

Comment:
can factor this out to avoid duplicating between here and loadView?

Reply:
done